pawnjester/More-recipesApp
# #154465272 write more tests

#### What does this PR do?
write more tests
#### Description of Task to be completed?
 * write more tests for the server side
#### How should this be manually tested?
Have the app working by:
 * run command `npm run test:dev` to run test
* run command `npm test` to see test coverage
 #### Any background context you want to provide?
 Nil
 #### What are the relevant pivotal tracker stories?
#154465272 write more tests
 #### Screenshots (if appropriate)
<img width="920" alt="screen shot 2018-01-19 at 1 13 18 pm" src="https://user-images.githubusercontent.com/26750279/35150415-936dffa8-fd1a-11e7-9ace-50001e106132.png">
